lib/vmpooler/api/v1.rb

```diff
@@ -66,8 +66,9 @@ def account_for_starting_vm(template, vm)
           backend.hget('vmpooler__token__' + request.env['HTTP_X_AUTH_TOKEN'], 'user')
         )
 
-        if config['vm_lifetime_auth'].to_i > 0
-          backend.hset('vmpooler__vm__' + vm, 'lifetime', config['vm_lifetime_auth'].to_i)
+        pool_lifetime_auth = pools.find {|p| p["name"] == template }['vm_lifetime_auth']
+        if (pool_lifetime_auth || config['vm_lifetime_auth']).to_i > 0
+          backend.hset('vmpooler__vm__' + vm, 'lifetime', (pool_lifetime_auth || config['vm_lifetime_auth']).to_i)
```

Comment:
You're calling `(pool_lifetime_auth || config['vm_lifetime_auth'])` enough that it might be worth moving that into an accessor or a helper method:

```ruby
# Stick this somewhere useful, maybe where `#pool_lifetime_auth` is defined?
# @return [Float, nil] the length of time a VM has been active
def auth_lifetime
  pool_lifetime_auth || config['vm_lifetime_auth']
end

if auth_lifetime.to_i > 0
  backend.hset('vmpooler__vm__' + vm, 'lifetime', auth_lifetime.to_i)
end
```

This is kind of boilerplate, but it's something to consider. I'm OK with merging this and then implementing something like what I've suggested in another PR.